backend/cms_app/models.py

```diff
@@ -0,0 +1,28 @@
+"""
+Models for the paris_1970 app.
+
+"""
+from django.db import models
+
+
+class Category(models.Model):
+    tag = models.CharField(max_length=20, help_text="Enter blog category")
+
+    def __str__(self):
+        return self.tag
+
+
+class Post(models.Model):
+    title = models.CharField(max_length=100)
```

Comment:
I don't think these tables are receiving any rows? I am seeing my posts being added to the cms built-in page table 
In the terminal: 
```
page = Page.objects.first()
page.get_page_title() 
```